Flask/app.py

```diff
@@ -0,0 +1,26 @@
+import flask
+
+from flask import render_template
+from flask import request, redirect
+
+app = flask.Flask(__name__)
+
+
+@app.route('/mypage/me')
+def me():
```

Comment:
nazwij tę funkcję tak, żeby informowała o tym co robi. me() nie daje programiście za dużo informacji, nawet jeśli wcześniej widział jak wygląda definiowanie route'ów we Flasku